### ISSUE TITLE
Need to separate affinity sections for worker and balancer

### ISSUE THREAD

**Issue by ksa-real** [manticoresoftware/manticoresearch-helm#18]
The `affinity` section is also used to apply anti-affinity. The issue is the same values are applied to workers and the balancer. This is incorrect as the balancer should be allowed to run on the same nodes with a worker.

The ask is to make a separate set of values for affinity and probably similar values: resources, nodeSelector, tolerations, affinity. So instead of mentioned fields at the top context, make the same fields inside the `worker` and `balancer` object:
```yaml
worker:
  affinity:
    podAntiAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
      - labelSelector:
          matchLabels:
            name: myrelease-manticore-worker
            app.kubernetes.io/instance: myrelease-manticore
            app.kubernetes.io/name: manticore
        topologyKey: "kubernetes.io/hostname"
balancer:
  resources:
    ...
```

**Comment by sanikolaev** [comment]
@ksa-real would you like to make a pull request with this change?

**Comment by ksa-real** [comment]
@sanikolaev Please review

**Comment by sanikolaev** [comment]
> @sanikolaev Please review

@djklim87 pls review

**Comment by djklim87** [comment]
Already implemented in 5.0.0.2
